tests/unit/test_codes.py

```diff
@@ -32,6 +32,11 @@ def test_codelist_from_csv_missing_file(tmp_path):
         codelist_from_csv(tmp_path / "no_file_here.csv", column="CodeID")
 
 
+def test_codelist_from_csv_missing_file_hint(tmp_path):
+    with pytest.raises(CodelistError, match="backslash"):
+        codelist_from_csv(tmp_path / "bad\\separator\\file.csv", column="CodeID")
```

Comment:
`codelist_from_csv()` would usually be called with a string which is then turns into a path.  I think it would make sense for the tests to do the same.

Reply:
Yes, good point. I think I should explicitly check that they accept both.